test/createFunction/createFunction.CSharp.test.ts

```diff
@@ -137,7 +137,7 @@ function addSuite(version: FuncVersion): void {
             );
```

Comment:
Seems like you're missing some changes you've done in other repos, like in package.json and .azure-pipelines

Reply:
Damn, I know I did them... I think I forgot to save the changes.  Need to turn auto-save back on lol